### PR TITLE
chore: clarify multiple watchSubscriptions

### PR DESCRIPTION
**Pull request by chris13524**

### CODE REVIEW

File: docs/specs/clients/notify/rpc-methods.md [hunk 1]
```diff
@@ -39,8 +39,7 @@ sequenceDiagram
 ```
 
 Notes:
-- Each client will call `wc_notifyWatchSubscriptions` separately with separate persistent keys kY
-- Recommended: persist kY to avoid multiple unnecessary response topics
+- Each client will call `wc_notifyWatchSubscriptions` to receive subscription updates for a blockchain account
```
Comment (Elyniss) [inline docs/specs/clients/notify/rpc-methods.md:42]:
```suggestion
- Generate and persist kY per req topic and blockchain account
- Each client will call `wc_notifyWatchSubscriptions` to receive subscription updates for a blockchain account
```

Comment (chris13524) [reply docs/specs/clients/notify/rpc-methods.md:42]: See latest version. I actually removed this detail from these bullet points in-favor of below in the method description.

File: docs/specs/clients/notify/rpc-methods.md [hunk 2]
```diff
@@ -39,19 +39,20 @@ sequenceDiagram
...
+- Notify server has an "watcher" timeout and will no longer send updates after 1 day of not calling `wc_notifyWatchSubscriptions`
   - Client must call `wc_notifyWatchSubscriptions` at least once a day to continue receiving updates
 
 ### wc_notifyWatchSubscriptions
 
 Watches for updates to subscriptions for an account. Each update will result in a `wc_notifySubscriptionsChanged` request to the client. Calling this will also immediately trigger a `wc_notifySubscriptionsChanged` with the current subscription state.
 
-If the same `watchSubscriptionsAuth.iss` is used as a previous request, the `wc_notifySubscriptionsChanged` update timeout will be reset. It is recommended that you re-use the same `iss` and client persistent private key for the same account on the same device.
+Calling this method will create a "watcher" on the Notify Server for the given account and `iss`. A symkey and topic is derived from `kY` (which is specified under Response below) and saved for future updates. This deriveed topic is used to send the method response, and to future updates to an account's subscriptions. Updates will stop and the watcher removed after the watcher timeout, which is 1 day.
```
Comment (Elyniss) [inline docs/specs/clients/notify/rpc-methods.md]:
```suggestion
Calling this method will create a "watcher" on the Notify Server for the given account and `iss`. A symkey and topic is derived from `kY` (which is specified under Response below) and saved for future updates. This derived topic is used to send the method response, and to future updates to an account's subscriptions. Updates will stop and the watcher removed after the watcher timeout, which is 1 day.
```

**Review by Elyniss — APPROVED** [review]
only typo, lgtm

File: docs/specs/clients/notify/rpc-methods.md (hunk 2 above)
Comment (chris13524) [reply docs/specs/clients/notify/rpc-methods.md]: Already fixed :D

**Pull request merged**